_posts/2015/2015-12-07-codemotion-madrid.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: post
+title: "Codemotion 2015 - One ePagees was there!"
+date: "2015-12-07 12:00:00"
+icon: search
+categories: conferences
+authors: ["Ángel García"]
+---
+The Codemotion conference took place in Madrid on November 28 and 29. ePages was present there. With more than 1900 software enthusiasts, Codemotion is the biggest IT event in Spain. This conference was sponsored by companies such as IBM, Microsoft, Arsys, Mozilla, Google... Two days fulled of talks from 12 different tracks, where each track
+focussed on a specific theme such as  Software Quality, Agile Methodologies, Clean Code, Testing, Big Data, Infrastructure,
+Scala, Java, Cassandra, MongoDB, Apache Drill, Selenium, Spark... Some of them have been broadcasted, and now you have the chance to watch them on [Youtube](http://2015.codemotion.es/streaming.html) (unfortunately only available in Spanish).
+
+It was a good opportunity to get inspiration and I really enjoyed the variety of talks, which allowed me to get to know new technologies and discover different points of view on how to deal with tricky situations in some of the most well-known technology companies. From my point of view, each talk had different levels of difficuly and quality, I think that most of the speakers were quite experts in the topic. I would like to highlight here some talks that I consider the most interesting.
```

Comment:
From my point of view, each talk had different levels of difficuly and quality, > Make a fullstop behind quality.
Start with a new sentence: I think that most of the ...